gst/nnstreamer/tensor_query/tensor_query_serversrc.c

```diff
@@ -304,6 +304,7 @@ gst_tensor_query_serversrc_change_state (GstElement * element,
     case GST_STATE_CHANGE_PLAYING_TO_PAUSED:
       src->playing = FALSE;
       gst_tensor_query_server_release_edge_handle (src->src_id);
+      src->configured = FALSE;
```

Comment:
Care to explain?

Reply:
Updated commit message.
It seems easier to understand to set caps again in `change_state` callback.
So the code has also been updated.
Thank you :)